forte/api/forte_python_module.cc

```diff
@@ -391,6 +391,8 @@ PYBIND11_MODULE(_forte, m) {
              "Set the map from state to the weights of all computed roots")
         .def("set_read_cwd_amps", &SADSRG::set_read_amps_cwd,
              "Set if reading amplitudes in the current directory or not")
+        .def("set_die_if_not_converged", &SADSRG::set_die_if_not_converged,
```

Comment:
I see what you are doing. I think that this kind of logic should live on the python side to make it more explicit. But for the moment I am OK with it living here.

Reply:
I agree. I moved the convergence test to python side.